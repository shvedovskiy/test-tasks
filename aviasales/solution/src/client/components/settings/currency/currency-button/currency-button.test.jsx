/* eslint-disable no-undef */
import React from 'react';

import CurrencyButton from './currency-button';


describe('<CurrencyButton />', () => {
  const props = {
    currencyName: 'Russian Rouble',
    checked: false,
    alias: 'RUB',
    handleChangeCurrency: jest.fn(),
  };
  const currencyButton = shallow(<CurrencyButton {...props} />);

  it('renders properly', () => {
    expect(currencyButton).toMatchSnapshot();
  });
});
