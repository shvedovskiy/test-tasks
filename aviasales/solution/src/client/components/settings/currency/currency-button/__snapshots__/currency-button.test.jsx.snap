// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurrencyButton /> renders properly 1`] = `
<styled__SwitchButton
  checked={false}
>
  <styled__Radio
    checked={false}
    id="Russian Rouble-currency"
    name="currency"
    onChange={[Function]}
    value="Russian Rouble"
  />
  <styled__RadioLabel
    htmlFor="Russian Rouble-currency"
  >
    RUB
  </styled__RadioLabel>
</styled__SwitchButton>
`;
